package.json

```diff
@@ -257,8 +257,9 @@
     "prop-types": "15.7.2",
     "rc-cascader": "1.0.1",
     "re-resizable": "^6.2.0",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
+    "react-beautiful-dnd": "13.0.0",
```

Comment:
was added as got new errors that this could not be resolved

Reply:
hmm, I think that one is hoisted from the `grafana/ui` package. Since the Transformations UI was located in the `grafana/ui` in the beginning the `react-beautiful-dnd` was added as a dependency to `grafana/data`. But looks like we did not move it to the root package when moving the transformations UI.

So we can remove it completely from `grafana/ui` and only have it in the root project.